labelbox/schema/annotation_import.py

```diff
@@ -141,6 +141,18 @@ def _create_from_bytes(cls, client, variables, query_str, file_name,
         files = {file_name: file_data}
         return client.execute(data=data, files=files)
 
+    @classmethod
+    def _get_ndjson_from_objects(cls, data, slug):
```

Comment:
nit: 
Instead of `data` and `slug`, can we rename to `objects` and `object_name`, which stays more consistent with the function name?